### ISSUE TITLE
SLCA performance with MAE pretrained ViT-B/16

### ISSUE THREAD

**Issue by TTangwj** [GengDavid/SLCA#4]
Hi,@GengDavid,great work!
I tried to apply SLCA on cifar100 and imagenet-r with MAE pretrained ViT-B/16,and I got  53.93 and 39.65 last-acc ,respectively,both are lower than result of Seq FT w/SL with MAE,as shown in fig.3

<img width="385" alt="Image" src="https://github.com/user-attachments/assets/1777e6f3-c03f-4e87-9a57-f8b5fe59a39b" />
Did I miss something?Looking forward to your reply,thanks!

**Comment by GengDavid** [comment]
Thanks for your interest! Fine-tuning MAE requires addtional modification such as layer-wise learning rate decay. If you need guidance, please refer to the [MAE official repo](https://github.com/facebookresearch/mae/blob/efb2a8062c206524e35e47d04501ed4f544c0ae8/main_finetune.py#L283) for the detail.

**Comment by TTangwj** [comment]
Thanks for your response! I will check it for more details.
